src/pages/workspaces/workspace/useDeleteWorkspaceState.ts

```diff
@@ -112,14 +112,14 @@ export const useDeleteWorkspaceState = (hookArgs: DeleteWorkspaceHookArgs): Dele
   })
 
   const hasApps = () => {
-    return (workspaceResources !== null && !_.isEmpty(workspaceResources.apps))
+    return (workspaceResources !== undefined && !_.isEmpty(workspaceResources.apps))
```

Comment:
FYI, there is a `_.isUndefined` method you can use if you find that nicer. I am completely ambivalent.